hydra/test_utils/launcher_common_tests.py

```diff
@@ -7,14 +7,13 @@
 from typing import Any, List, Optional, Set
 
 import pytest
-from omegaconf import DictConfig, OmegaConf
-
 from hydra import TaskFunction
 from hydra.test_utils.test_utils import (
     TSweepRunner,
     integration_test,
     verify_dir_outputs,
 )
+from omegaconf import DictConfig, OmegaConf
```

Comment:
odd, was this not sorted?

Reply:
iirc it looks like this after sorting :(

Reply:
check the contributing document to see how to run isort for plugins through nox.